src/components/util/noop.js

```diff
@@ -0,0 +1,2 @@
+const noop = () => {};
```

Comment:
`() => {}` is es6 convention for a noop. Not sure we need a utility for this, but I'm not against it either

Reply:
hmmm so I agree in a sense... but usually when I do noop's, I do them inline. The fact that you broke it out into a variable, was a signal to me that we'd have multiple noop variables in different files, and I'd just break it out into a utility. I guess my follow up question would be... do we need variables for this?

Reply:
probably not - if I stored a noop in a var, I probably shouldn't have

Reply:
yeah. You have noops defined in both this story, and the popover component. I'll wipe them away (if that's what you want)

Reply:
actually... the subsequent one in Popover may have been Nico.